homework/grade2/Tom-grade-2.py

```diff
@@ -0,0 +1,41 @@
+# homework for grade 2
+def stringornumber():
+
+    check = input('Enter a number or string: ')
+
+    if check.isalpha():
+        print('{} is a string'.format(check))
+    elif check.isnumeric():
+        print('{} is a number'.format(str(check)))
+    else:
+        print('{} characters and numbers'.format(check))
+
+def doathing(number):
```

Comment:
Not taught you this yet, but in python3 it's good practice to follow pep8 coding standards. So function names use underscores, eg: `do_a_thing` and class names are Camel case, eg ClassThatDoesStuff.